### ISSUE TITLE
Tracking du modèle fusion via MLFlow

### ISSUE THREAD

**Issue by JoffreyLGT** [JoffreyLGT/e-commerce-mlops#116]
Mettre en place Airflow sur le script du modèle fusionné.

**Comment by JoffreyLGT** [comment]
I'm starting to work on this. It's gonna be a bit tricky since I will have to use image and text model served by MLFlow and currently in staging or production state.

**Comment by JoffreyLGT** [comment]
Quick update

I faced many issues regarding regarding text_preprocessor. It has been a pain to diagnose MLFLOW issues when serving the model locally. Long story short: when moving the mlruns folder somewhere else and serving the model locally, it's impossible to load a pickle.

I had to change the way we store the preprocessor and use JSON files instead.

All the modifications done are so heavy that I am forced to do #131 at the same time, otherwise the project will not work anymore.
